### ISSUE TITLE
It does not work properly under "nuxt": "^3.7.4"

### ISSUE THREAD

**Issue by popdo** [yisibell/nuxt-svg-icons#3]
It does not work properly under "nuxt": "^3.7.4" when imported into modules.

An error occurs when running.

```
client:268  WebSocket connection to 'ws://localhost:5178/_nuxt/' failed: 
setupWebSocket @ client:268
(anonymous) @ client:262
client:268  Uncaught (in promise) DOMException: Failed to construct 'WebSocket': The URL 'ws://localhost:undefined/_nuxt/' is invalid.
```

**Comment by yisibell** [comment]
> It does not work properly under "nuxt": "^3.7.4" when imported into modules.
> 
> An error occurs when running.
> 
> ```
> client:268  WebSocket connection to 'ws://localhost:5178/_nuxt/' failed: 
> setupWebSocket @ client:268
> (anonymous) @ client:262
> client:268  Uncaught (in promise) DOMException: Failed to construct 'WebSocket': The URL 'ws://localhost:undefined/_nuxt/' is invalid.
> ```

It seems to be `vite hmr` 's problem. Under investigation.

**Comment by m7rlin** [comment]
It happens after you add 'nuxt-svg-icons' module.

**Comment by yisibell** [comment]
Now, it's fixed in `v1.23.1`.
